scripts/init-submodules-no-riscv-tools-nolog.sh

```diff
@@ -20,43 +20,41 @@ fi
 DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 CHIPYARD_DIR="$(dirname "$DIR")"
 
-# Ignore toolchain submodules
 cd "$CHIPYARD_DIR"
-for name in toolchains/*-tools/*/ ; do
-    git config submodule."${name%/}".update none
-done
-git config submodule.toolchains/libgloss.update none
-git config submodule.toolchains/qemu.update none
 
-# Don't automatically initialize generators with big submodules (e.g. linux source)
-git config submodule.generators/sha3.update none
-git config submodule.generators/gemmini.update none
+(
+    # Blacklist of submodules to initially skip:
```

Comment:
Can we change this to skiplist or blocklist?

Reply:
Sure, I went with "blocklist".